app/_how-tos/store-keyring-in-hashicorp-vault.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Store Keyring data in a HashiCorp Vault
+content_type: how_to
+related_resources:
+  - text: Keyring
+    url: /gateway/entities/keyring/
+  - text: Configure HashiCorp Vault as a vault backend
+    url: /how-to/configure-hashicorp-vault-as-a-vault-backend
```

Comment:
```suggestion
related_resources:
  - text: Keyring
    url: /gateway/entities/keyring/
  - text: Configure HashiCorp Vault as a vault backend
    url: /how-to/configure-hashicorp-vault-as-a-vault-backend/
```
I'm not sure if this is the fix, but for some reason, these related links aren't in the info box in the rendered page